### ISSUE TITLE
Lap marker when selecting 2 activites

### ISSUE THREAD

**Issue by tsofron** [gerhardol/trails#34]
Hello Gerhard,

any chance you might add the lap marker on the map, when selecting 2 activities? It would be more intuitive to see where the gps went crazy...I know it's an old project, but still would be useful...
Edit - i mean the actual split value (1k, 2k, etc...)

Thanks,
Tudor

**Comment by gerhardol** [comment]
I still use Trails most days so it is active for me - but I have not added anything public for a while.
The lapmarker is there (for Splits trail), for the summary select in the list. You can ctrl-click to see several laps

I am not too keen into adding a popup on the map, it is a pain to do that and avoid interfering with ST controls.

**Comment by tsofron** [comment]
Yeah - i finally figured out how to make the map providers work and now using Trails / Overlay is a real pleasure :-) - don't really know why i haven't been using it before (ok, probably i've been not comparing tracks :D, so probably that's why). 
In this case for example - 
![image](https://user-images.githubusercontent.com/72147998/95939782-8bdb2980-0de5-11eb-8d05-caadd018bc66.png)
having the lap number would be really helpful - so you would see the big picture. If it would be easier, same functionality in the Overlay plugin would also do the trick, as i would just want to see where the lap actually are:
![image](https://user-images.githubusercontent.com/72147998/95939947-eb393980-0de5-11eb-9acc-1bb27426de9b.png)

I'm asking you this as you've probably got more experience with the plugin :D and drawing things, and it would be definitely easier for you to get the code right from the beginning :-)

Thanks,
Tudor

**Comment by gerhardol** [comment]
But you have the information in the table, just not on the map
Add more info may get the mapview messy when there are many activity results

(I had almost forgot about Overlay plugin...)

![image](https://user-images.githubusercontent.com/6248932/96191761-0a9aa880-0f45-11eb-86ec-5972fd86a654.png)

**Comment by tsofron** [comment]
I know :-) - but it's easier to visualize on the map actually where the gps went haywire :-) For me the compare scenario is pretty simple - i'm just comparing 2- at most 3 tracks...
